processing/src/main/java/io/druid/query/QueryInterruptedException.java

```diff
@@ -118,6 +119,8 @@ private static String getErrorCodeFromThrowable(Throwable e)
       return QUERY_CANCELLED;
     } else if (e instanceof TimeoutException) {
       return QUERY_TIMEOUT;
+    } else if (e instanceof ResourceLimitExceededException) {
```

Comment:
If we're not careful we're going to have something that looks like a collection of http status codes in a big nasty switch statement here. If theses ever get expanded with more druid exceptions then we'll probably want to rethink this workflow.